sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/TokenTest.java

```diff
@@ -185,7 +187,7 @@ public void canReadTokenFromCacheWhenTokenValid() {
                 .withDbAccount("dbAccount")
                 .withDbCollectionName("collection")
                 .withToken(FAKE_TOKEN));
-        when(SharedPreferencesManager.getString(READONLY_PARTITION_NAME)).thenReturn(tokenResult);
+        when(SharedPreferencesManager.getString(PREFERENCE_PARTITION_PREFIX + READONLY_PARTITION_NAME)).thenReturn(tokenResult);

```

Comment:
what is `PREFERENCE_PARTITION_NAMES`?

Reply:
READONDLY partition is readonly partition

Reply:
Please read description of the issue.